x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/OptimizerRulesTests.java

```diff
@@ -227,17 +222,6 @@ public void testDualEqualsConjunction() {
         assertEquals(FALSE, exp);
     }
 
-    // a <=> 1 AND a <=> 2 -> FALSE
```

Comment:
These test cases were just copied over from the QL version of the optimizer rules.